examples/pylab_examples/set_and_get.py

```diff
@@ -1,24 +1,24 @@
 """
 
-MATLAB and pylab allow you to use setp and get to set and get
+MATLAB and pyplot allow you to use setp and get to set and get
```

Comment:
Let's go farther here: I don't think we need to refer to Matlab at all, and although "get" is an alias for "getp", I think it would be better to encourage use of the symmetric pair, setp and getp.  These are indeed what is used in the example.